linux/ibus-keyman/tests/testfixture.cpp

```diff
@@ -88,16 +88,6 @@ ibus_keyman_tests_fixture_tear_down(IBusKeymanTestsFixture *fixture, gconstpoint
   delete data;
 }
 
-static void
-set_caps_lock_state(IBusKeymanTestsFixture *fixture, bool caps_lock_on) {
-  set_capslock_indicator((guint32)caps_lock_on);
-}
-
-static bool
-get_caps_lock_state(IBusKeymanTestsFixture *fixture) {
-  return get_capslock_indicator();
-}
-
```

Comment:
> Also some cleanup.

Sorry to ask for extra work. Could we split this to a separate PR, so this PR only has `--remote-debug` changes?

Reply:
#13246